package.json

```diff
@@ -46,11 +48,13 @@
     "gruntfile-editor": "^0.2.0",
     "inquirer": "^0.8.0",
     "isbinaryfile": "^2.0.0",
+    "istextorbinary": "^1.0.2",
```

Comment:
@Toilal Yeah, I guess it doesn't matter. My only concern is that this one requires us to read in the whole file and then detect whether it's binary. `isbinaryfile` only read the required bytes and should be much faster.